llvm/unittests/Support/Path.cpp

```diff
@@ -1296,6 +1296,39 @@ TEST_F(FileSystemTest, UTF8ToUTF16DirectoryIteration) {
 }
 #endif
 
+TEST_F(FileSystemTest, OpenDirectoryAsFileForRead) {
+  ASSERT_NO_ERROR(fs::create_directory(Twine(TestDirectory)));
```

Comment:
The code here does not include code to remove this directory. Perhaps it should?

Reply:
The use of `TestDirectory` here does not seem consistent with the other uses of the `TestDirectory` that is shared between tests.

https://github.com/llvm/llvm-project/blob/b667596e78d16c613ef0b6d2d16d1ea1c3493f78/llvm/unittests/Support/Path.cpp#L665-L667

It seems that `TestDirectory` is created (and removed) by the test harness.

It seems we don't need to create/remove a directory here?